CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
 # Changelog
+All notable changes to this project will be documented in this file.
+
+This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+### Fixed
+* [#1001][]: Fix complex number encoding with negative imaginary part. Thanks to @hemantjadon.
+* [#1003][]: Fix float32 encoding.
```

Comment:
Something more specific here? It's not that float32 was entirely broken; only that the preciseness was lost.

```suggestion
* [#1003][]: JSON: Fix inaccurate precision when encoding float32.
```